packages/alignments/src/BamAdapter/BamAdapter.test.js

```diff
@@ -57,3 +57,50 @@ test('test usage of BamSlightlyLazyFeature toJSON (used in the drawer widget)',
   expect(f.end).toBe(102)
   expect(f.mismatches).not.toBeTruthy()
 })
+
+test('test usage of getMultiRegion stats, adapter can generate a domain', async () => {
+  const adapter = new Adapter({
+    bamLocation: {
+      localPath: require.resolve('../../test_data/volvox-sorted.bam'),
+    },
+    index: {
+      location: {
+        localPath: require.resolve('../../test_data/volvox-sorted.bam.bai'),
+      },
+      indexType: 'BAI',
+    },
+  })
+
+  const stats = await adapter.getMultiRegionStats(
+    [
+      {
+        refName: 'ctgA',
+        start: 0,
+        end: 100,
+        parentRegion: {
+          refName: 'ctgA',
+          start: 0,
+          end: 10000,
+        },
+      },
+    ],
+    {
+      opts: {
+        signal: {
+          aborted: false,
+          onabort: null,
+        },
+        bpPerPx: 0.2,
+      },
+    },
+    100,
+  )
+
+  expect(stats).toEqual(
+    expect.objectContaining({
+      scoreMin: 0,
+      scoreMax: expect.any(Number),
```

Comment:
Can you make the scoreMax the actual value that it should be e.g. the actual maximum value over this range?